Twig/Extension/SonataAdminExtension.php

```diff
@@ -437,6 +437,12 @@ protected function getTemplate(
         try {
             $template = $environment->loadTemplate($templateName);
         } catch (\Twig_Error_Loader $e) {
+            @trigger_error(
+                'Relying on default template loading on field template loading exception '.
+                'is deprecated since 3.x and will be removed in 4.0, '.
```

Comment:
Much better! :+1: 

I would improve this way:

`is deprecated since 3.x and will be removed in 4.0.`

`A \Twig_Error_Loader exception will be thrown instead.`